storybook/storybook-docs/src/typography.stories.mdx

```diff
@@ -166,14 +164,15 @@ Vermijd het onderstrepen van tekst.
 Onderstreep alleen interactieve links.
 
 De lijndikte is 1/12 van de tekstgrootte, dus 2 pixels bij een standaard paragraaf van 24.
-De afstand tussen de onderstreping en de letterlijn is iets vergroot.
-Rond staartletters wordt de streep onderbroken voor betere leesbaarheid.
+De afstand tussen de onderstreping en de letterlijn is 3/8 van de tekstgrootte –
+meer dan we in onderstaande voorbeeld kunnen laten zien.
```

Comment:
This implies a fluid styling imo, which currently isn't the case. The underline thickness is a static 0.125rem, and the distance between baseline and underline 0.375rem.

Reply:
Ah yes. Adjusted.